include/service/entities/template_service.js

```diff
@@ -294,27 +313,46 @@ module.exports = function(pb) {
      */
     TemplateService.prototype.getTemplateContentsByPriority = function(relativePath, cb) {
         var self = this;
-        
+
         //build set of paths to search through
-        var hintedTheme   = this.getTheme();
-        var paths         = [];
-        if (hintedTheme) {
-            paths.push(TemplateService.getCustomPath(this.getTheme(), relativePath));
+        var paths = [];
+        var themePath = null;
+        var hintedTheme = this.getTheme();
+        if (hintedTheme && !TemplateService.isTemplateBlacklisted(hintedTheme, relativePath)) {
```

Comment:
The blacklisting was introduced because the profiler showed we spent a lot of time looking for templates in other plugins.  We did this on each request regardless of whether or not we had already tried it.  The blacklist reduces the number of times we ever look at disk for a theme and path combo.